features/convergedcloud/pkg.include

```diff
@@ -1,14 +1,15 @@
 apparmor
-open-vm-tools
+arptables
 containerd
+curl
 docker.io
+ebtables
 ethtool
-socat
-curl
-logrotate
+gnupg
 jq
-nfs-common
+logrotate
 netcat-openbsd
-ebtables
-arptables
+nfs-common
+open-vm-tools
+socat
```

Comment:
Are you sure that package is required in this context or is it intended to help support in specific circumstances?

Reply:
This list was only sorted for better readability, so I'd do this in #17